src/dotnet/APIView/APIViewWeb/Pages/Assemblies/RequestedReviews.cshtml

```diff
@@ -0,0 +1,142 @@
+                        {
+                            @foreach (var review in Model.ApprovedReviews)
+                            {
+                                var truncationIndex = @Math.Min(@review.DisplayName.Length, 100);
+                                <tr class="package-data-row">
+                                    <td class="align-middle pl-4 pt-0 pb-0">
+                                        @if (review.Language != null)
+                                        {
+                                            string iconClassName = "icon-" + review.GetLanguageCssSafeName();
+                                            @if (!string.IsNullOrEmpty(review.LanguageVariant) && review.LanguageVariant.ToLower() != "default")
+                                            {
+                                                iconClassName += "-" + @review.LanguageVariant.ToLower();
+                                            }
+                                            <span role="img" class="mx-1 icon icon-language @iconClassName" aria-label="@review.Language @review.LanguageVariant"></span>
+                                        }
+                                        <a class="review-name align-middle" asp-page="./Review" asp-route-id="@review.ReviewId">@review.DisplayName.Substring(0, @truncationIndex)</a>
+                                        @if (review.IsApproved == true)
+                                        {
+                                            <i class="fas fa-check-circle text-success ml-2"></i>
+                                        }
+                                    </td>
+                                    <td class="align-middle border-left pl-3">
+                                        <a username="@review.Author">@review.Author</a>
+                                    </td>
+                                    <td class="align-middle border-left pl-3">
+                                        <span data-placement="left" data-toggle="tooltip" title="@review.LastUpdated" date="@review.LastUpdated.ToLocalTime()"></span>
+                                    </td>
+                                    <td class="align-middle border-left pl-3">
+                                        <span>@review.FilterType.ToString()</span>
+                                    </td>
+                                    <td class="align-middle border-left pl-3">
+                                        <span data-placement="left" data-toggle="tooltip" title="@review.LastUpdated" date="@review.approvalDate.ToLocalTime()"></span>
+                                    </td>
+                                    <td class="align-middle border-left pl-3">
+                                        <span>
+                                            @foreach(var approver in review.Revisions.First().Approvers) {
+                                                 <a username="@approver"> @approver </a>
```

Comment:
Random thought - do we change the links where we link to a user throughout APIView to instead be a link to their user profile page (if one exists, and assuming we build a read-only version of their profile page). This might be useful in the sense that it shows me their email address, as well as a link to their GitHub profile. Something for @chidozieononiwu to consider.

Reply:
Isn't that what happens already? Am I missing something?

Reply:
When the user profiles pages gets pushed through as we are doing for #4164 currently, it should be doing this